test/webserver.mjs

```diff
@@ -330,6 +331,16 @@ class WebServer {
       "Content-Range",
       `bytes ${start}-${end - 1}/${fileSize}`
     );
+
+    // Support test in `test/unit/network_spec.js`.
+    switch (searchParams.get("test-network-break-ranges")) {
```

Comment:
Don't you need the Cache-Control header for the non-Range request to avoid Chrome's caching issue? See the Cache-Control header and comment in the redirectHandler that I added recently.

Reply:
The unit-tests seem to pass as-is, both locally and here in the PR, so I guess it's fine without that.